core/storage/user/gae_models.py

```diff
@@ -925,6 +925,27 @@ def construct_model_id(cls, user_id, skill_id):
         """
         return '%s.%s' % (user_id, skill_id)
 
+    @classmethod
+    def export_data(cls, user_id):
+        """Exports the data from UserSkillMasteryModel
+        into dict format for Takeout.
+
+        Args:
+            user_id: str. The ID of the user whose data should be exported.
+
+        Returns:
+            dict. Dictionary of the data from UserSkillMasteryModel.
+        """
+
+        user_data = dict()
+        mastery_models = cls.get_all().filter(cls.user_id == user_id).fetch()
+
+        for mastery_model in mastery_models:
+            mastery_model_skill_id = mastery_model.key.string_id()
```

Comment:
I mentioned before to just use `.id` instead of `.key.string_id()` ...

Reply:
Hey @seanlip I’m a bit confused. https://github.com/oppia/oppia/pull/7264#discussion_r310303338 here you indicated you did not want the model ID you wanted the skill ID. If we use ‘.id’ then this will give the model id rather than skill id

Reply:
@seanlip friendly ping :)

Reply:
Ah -- what I meant was, don't use .key.string_id() in general, use .id instead.

But in this case I don't believe that .key.string_id() gives you the skill ID either (and you are indeed correct that we don't want the model ID, we want the skill ID).

Reply:
Done